keyboards/clickety_split/leeloo/keymaps/default/config.h

```diff
@@ -0,0 +1,31 @@
+/* Copyright 2022 Clickety Split Ltd.
+ *                https://clicketysplit.ca
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+// Select hand configuration
+#define MASTER_LEFT
```

Comment:
This is the default behavior, and is not needed. 
```suggestion
// #define MASTER_LEFT
```